### PR TITLE
Do not include itext transaltion for missing media

### LINKED ISSUE #32: 2 translated media labels outputs a 3rd (default) language
screen shot 2015-04-14 at 3 26 35 pm

Form source here: https://docs.google.com/spreadsheets/d/1HlxpC4ORzor94WXONUqJXbrrlrsSQAEXdGsiAviM4lU/edit?usp=sharing

The default language has no valid audio src (because it doesn't exist in the XLSForm).

XForm itext output is:

``` xml
<itext>
    <translation default="true()" lang="default">
        .....
    </translation>
    <translation lang="Russian">
        ......      
    </translation>
    <translation lang="Kyrgyz">
       .......
    </translation>
</itext>
```

But expected itext output is:

``` xml
<itext>
    <translation lang="Russian">
        .....
    </translation>
    <translation lang="Kyrgyz">
       .......
    </translation>
</itext>


```

### PR DESCRIPTION
**Pull request by ukanga**
Fix #32

### CODE REVIEW

File: pyxform/tests_v1/test_translations.py [hunk 1]
```diff
@@ -35,3 +37,38 @@ def test_langs(self):
...
+        |         | bt7nj36   | 0    | No    | Нет            | Нет (ky)      |
+        |         | bt7nj36   | 1    | Yes   | Да             | Да (ky)       |
+        """
+        self.assertPyxformXform(
+            name="multi_language_form",
+            id_string="multi_language_form",
+            md=xform_md,
+            errored=False,
+            debug=False,
+            itext__contains=[
```
Comment (moshthepitt) [inline pyxform/tests_v1/test_translations.py]: Should we check that it also includes `Kyrgyz`?

Comment (ukanga) [reply pyxform/tests_v1/test_translations.py]: Not really, this issue is about not having a valid audo src in the media label, `<value form="audio">jr://audio/-</value>` should not be part of the XForm XML.

Comment (ukanga) [reply pyxform/tests_v1/test_translations.py]: But I will add it.

Comment (ukanga) [reply pyxform/tests_v1/test_translations.py]: Done

**Pull request merged**